test/extended/networking/multicast.go

```diff
@@ -167,5 +168,16 @@ func launchTestMulticastPod(f *e2e.Framework, nodeName string, podName string) (
 		podIP = pod.Status.PodIP
 		return (podIP != "" && pod.Status.Phase != kapiv1.PodPending), nil
 	})
+
+	if err != nil {
+		// Append error logs
+		logs, logErr := e2e.GetPodLogs(f.ClientSet, f.Namespace.Name, podName, fmt.Sprintf("%s-pod", podName))
+		if logErr != nil {
+			e2e.Failf("Error getting container logs: %s", logErr)
+		}
+
+		err = errors.New(err.Error() + logs)
```

Comment:
hm... what does that end up looking like? (Don't you at least need a space or a newline between the original error and the logs?)

You could alternatively call e2e.Logf() to print the logs separately from returning the error...

Reply:
I think it might be useful to have the container log added to the error string.